CONTRIBUTING.md

```diff
@@ -1,3 +1,20 @@
+## Steps for Contributing
+1. Create a new branch off of up-to-date master
+2. Make your changes.  If you are adding an endpoint or resource to the API,
+follow "Adding an API Resource"
+3. Make sure your code follows the "Code Standards" section of this document
+4. Use "git rebase -i origin/master" to squash your commits into one or a few. Follow commit message guidelines.
+5. Push your feature branch and make sure Travis CI passes
+6. Submit a pull request to master and tag at least one reviewer
```

Comment:
Might also want to rework these last two steps if we want this document to apply to external contributors (CI won't run on a fork until a PR is opened, for example)